app/views/service_orders/partial/_event.html.slim

```diff
@@ -0,0 +1,21 @@
+h2.title.underline
+  | Special Events
+.intro
+  | Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old.
+.container
+  - (1..2).each do |i|
+    .row
+      - event_arr = [["Move to our new home", "home"], ["Vietnam trip", "plane"], ["Get our new car", "car"], ["Adam got his dream job", "laptop"], ["Eve honored phd", "graduation-cap"], ["Attended Brazil Worldcup", "futbol-o"]]
```

Comment:
Could we do the more readable way, like:

``` 
event_list = 
[
  {
    name: "Name 1",
    icon: "home"        
  },
  {
    name: "Name 2",
    icon: "home"        
  },  
],
[
...
]
```

Then loop each array inside ```event_arr```, and loop for each